### ISSUE TITLE
Compiling DART fails when MPI shared windows are disabled

### ISSUE THREAD

**Issue by fuerlinger** [dash-project/dash#182]
DART does not compile when `DART_MPI_DISABLE_SHARED_WINDOWS` is defined.

````
dart_globmem.c: In function ‘dart_team_memalloc_aligned’:
dart_globmem.c:286:10: error: ‘baseptr_set’ undeclared (first use in this function)
     free(baseptr_set);
...
````

**Comment by fuchsto** [comment]
@fuerlinger Curious ... I already fixed that some weeks ago. Did this fail on a fresh clone of the development branch?

**Comment by fuerlinger** [comment]
Yes, it did for me - I used the manual Makefiles but that should set the same preprocessor defines as the CMake build so that should not matter.

**Comment by fuchsto** [comment]
Ok, looks like I fixed that in an unmerged branch, shared windows are still enabled for build "Minimal" on development:

https://github.com/dash-project/dash/blob/development/dash/scripts/dash-ci-deploy.sh

**Comment by fuchsto** [comment]
@fuerlinger Fixed in your PR #183
